lib/app.js

```diff
@@ -125,7 +91,6 @@ export default class TrimetWebapp extends Component {
       <ResponsiveWebapp
         desktopView={desktopView}
         mobileView={mobileView}
-        LegIcon={LegIcon}
```

Comment:
Sorry, I almost missed this. `LegIcon` is still needed, it is used in the Print view.